### PR TITLE
[NEXUS-5420] Prepend base URL to all emails

### PR DESCRIPTION
**Pull request by adreghiciu**
Reopened: See sonatype/nexus/pull/809 for details

### CODE REVIEW

**Comment by nabcos** [comment]
+1

**Comment by adreghiciu** [comment]
CI :+1: https://bamboo.zion.sonatype.com/browse/NXO-OSSF36-1

File: nexus-core/src/main/java/org/sonatype/nexus/email/DefaultNexusEmailer.java [hunk 1]
```diff
@@ -105,10 +111,48 @@ public MailRequestStatus sendMail( MailRequest request )
...
+        final MailType mailType = eMailer.getMailTypeSource().getMailType( request.getMailTypeId() );
+
+        final StringBuilder messageBody = new StringBuilder().append( "Message from: " );
+
+        if ( mailType != null && mailType.isBodyIsHtml() )
+        {
+            messageBody
+                .append(
+                    StringUtils.isNotBlank( baseNexusUrl )
+                        ? String.format( "<a href=\"%s\"/>", baseNexusUrl )
```
Comment (jdillon) [inline nexus-core/src/main/java/org/sonatype/nexus/email/DefaultNexusEmailer.java:135]: doh!

this won't render anything useful at all :-(

Probably should be closer to:

```
String.format( "<a href=\"%s\">%s</a>", baseNexusUrl, baseNexusUrl )
```

Comment (kellyrob99) [inline nexus-core/src/main/java/org/sonatype/nexus/email/DefaultNexusEmailer.java:135]: Bingo! Deployed this and works as expected.

**Pull request merged**